flight_phase_files/HALO_RF05_20200128_info.yaml

```diff
@@ -19,59 +19,125 @@ remarks:
+  irregularities: []
+  segment_id: HALO-0128_cb1
+  start: 2020-01-28 16:17:27
+  end: 2020-01-28 16:31:33
+- kinds:
+  - circle
+  name: Circle 2
+  irregularities: []
+  segment_id: HALO-0128_c2
+  start: 2020-01-28 16:31:33
+  end: 2020-01-28 17:28:35
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 2 and circle 3
+  irregularities: []
+  segment_id: HALO-0128_cb2
+  start: 2020-01-28 17:28:35
+  end: 2020-01-28 17:38:40
+- kinds:
+  - circle
+  name: Circle 3
+  irregularities:
+  - Circle starts 4 minutes (instead of regularly 1 minute) before the first dropsonde.
+  segment_id: HALO-0128_c3
+  start: 2020-01-28 17:38:40
+  end: 2020-01-28 18:37:38
+  good_dropsondes: 11
+- kinds:
+  - circling
+  name: Circling 1
+  irregularities: []
+  segment_id: HALO-0128_o1
+  start: 2020-01-28 15:20:54
+  end: 2020-01-28 18:37:38
+  good_dropsondes: 35
+- kinds:
+  - straight_leg
+  name: Outbound leg towards NTAS
+  irregularities: []
+  segment_id: HALO-0128_sl1
   start: 2020-01-28 18:41:47
   end: 2020-01-28 19:01:46
-- kind: radar_calibration_tilted
-  name: radar_cal_tilted
+- kinds:
+  - radar_calibration_tilted
+  name: Radar calibration
+  irregularities: []
+  segment_id: HALO-0128_rct1
   start: 2020-01-28 19:07:44
   end: 2020-01-28 19:25:26
-- kind: straight_leg
-  name: leg2_inbound_NTAS
+- kinds:
+  - straight_leg
+  name: Inbound leg from NTAS
+  irregularities: []
+  segment_id: HALO-0128_sl2
   start: 2020-01-28 19:29:47
   end: 2020-01-28 19:50:47
-- kind: circle
-  name: c4
-  start: 2020-01-28 19:57:50
-  end: 2020-01-28 20:53:13
-- kind: circle_break
-  name: cb4to5
-  start: 2020-01-28 20:53:13
-  end: 2020-01-28 21:13:08
-- kind: circle
-  name: c5
-  start: 2020-01-28 21:13:08
-  end: 2020-01-28 22:07:21
-- kind: circle_break
-  name: cb5to6
-  start: 2020-01-28 22:07:21
-  end: 2020-01-28 22:29:35
-- kind: circle
-  name: c6
-  start: 2020-01-28 22:29:35
-  end: 2020-01-28 23:23:36
-- kind: lidar_calibration
-  name: lidar_cal
+- kinds:
+  - circle
+  name: Circle 4
+  irregularities: []
+  segment_id: HALO-0128_c4
+  start: 2020-01-28 19:56:50
+  end: 2020-01-28 20:56:39
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 4 and circle 5
+  irregularities: []
+  segment_id: HALO-0128_cb3
+  start: 2020-01-28 20:56:39
+  end: 2020-01-28 21:12:08
+- kinds:
+  - circle
+  name: Circle 5
+  irregularities: []
+  segment_id: HALO-0128_c5
+  start: 2020-01-28 21:12:08
+  end: 2020-01-28 22:11:43
+  good_dropsondes: 12
+- kinds:
+  - circle_break
+  name: Break between circle 4 and circle 5
+  irregularities: []
+  segment_id: HALO-0128_cb4
+  start: 2020-01-28 22:11:43
+  end: 2020-01-28 22:24:35
+- kinds:
+  - circle
+  name: Circle 6
+  irregularities:
+  - Circle starts 5 minutes (instead of regularly 1 minute) before the first dropsonde.
+  segment_id: HALO-0128_c6
+  start: 2020-01-28 22:24:35
+  end: 2020-01-28 23:24:05
+  good_dropsondes: 12
```

Comment:
There were 13 sondes during this circle.

Reply:
Good point. According to Geet the last two sondes were launched one after another, to check "variability from measurement". I included an irregularity for this circle.